### PR TITLE
[Task 1862941] navigate a specific storage account to browser storage explorer

### PR DESCRIPTION
**Pull request by shenqianjin**
[AB#1862941](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1862941)

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/OpenStorageBrowserAction.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import com.microsoft.tooling.msservices.serviceexplorer.AzureIconSymbol;
+import com.microsoft.tooling.msservices.serviceexplorer.Node;
+import com.microsoft.tooling.msservices.serviceexplorer.NodeActionEvent;
+import com.microsoft.tooling.msservices.serviceexplorer.NodeActionListener;
+import com.microsoft.tooling.msservices.serviceexplorer.azure.storage.StorageNode;
+
+@Name(OpenStorageBrowserAction.ACTION_NAME)
+public class OpenStorageBrowserAction extends NodeActionListener {
+
+    public static final String ACTION_NAME = "Open Storage Browser";
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/OpenStorageBrowserAction.java]: private

**Review by andxu — DISMISSED** [review]

**Comment by andxu** [comment]
LGTM

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/OpenStorageBrowserAction.java (hunk 1 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/OpenStorageBrowserAction.java]: protected at least. private will cause compile error.

Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/OpenStorageBrowserAction.java]: and updated.

**Comment by andxu** [comment]
rebase?

**Review by andxu — DISMISSED** [review]
LGTM

**Pull request merged**